packages/eslint-config-eslint/base.js

```diff
@@ -262,7 +261,7 @@ const jsConfigs = [js.configs.recommended, {
 }];
 
 // extends eslint-plugin-jsdoc's recommended config
-const jsdocConfigs = [jsdoc.configs.recommended, {
+const jsdocConfigs = [jsdoc.configs["flat/recommended"], {
```

Comment:
Will it be a breaking change?

Reply:
The new version of the plugin (^46.5.1) officially requires [Node.js >= 16](https://github.com/gajus/eslint-plugin-jsdoc/blob/v46.5.1/package.json#L70), but [so did](https://github.com/gajus/eslint-plugin-jsdoc/blob/v46.2.5/package.json#L68) the previous version (^46.2.5). So I guess we are not losing any compatibility with this change?

Reply:
I think this is not a breaking change in `eslint-config-eslint`. The `flat/recommended` config is the same as the `recommended` config, except that it's in the new format - `plugins` contains the plugin object itself instead of the string reference `"jsdoc"`, which we were manually translating on line 12.

```js
const assert = require("assert");
const jsdoc = require("eslint-plugin-jsdoc");

jsdoc.configs.recommended.plugins = { jsdoc }; // line 12 before this change
assert.deepStrictEqual(
    jsdoc.configs.recommended, // config object we were using before this change
    jsdoc.configs["flat/recommended"] // config object we are using after this change
);
console.log("ok");
```

Reply:
> The new version of the plugin (^46.5.1) officially requires [Node.js >= 16](https://github.com/gajus/eslint-plugin-jsdoc/blob/v46.5.1/package.json#L70), but [so did](https://github.com/gajus/eslint-plugin-jsdoc/blob/v46.2.5/package.json#L68) the previous version (^46.2.5). So I guess we are not losing any compatibility with this change?

I think it should be okay as this package already requires Node >=16.0.0:

https://github.com/eslint/eslint/blob/926a28684282aeec37680bbc52a66973b8055f54/packages/eslint-config-eslint/package.json#L43-L45

Not sure if there's a semantic difference between >=16 and >=16.0.0, though.